packages/vite/src/node/optimizer/optimizer.ts

```diff
@@ -202,7 +208,9 @@ async function createDepsOptimizer(
         try {
```

Comment:
Maybe we should wrap this callback and await that instead of just awaiting this part?